packages/react-aria-components/src/ComboBox.tsx

```diff
@@ -24,20 +24,25 @@ import {useCollection} from './Collection';
 
 export interface ComboBoxRenderProps {
   /**
-   * Whether the combobox is focused, either via a mouse or keyboard.
-   * @selector [data-focused]
```

Comment:
Ugh I'm having a hard time tracking down the discussion which decided that the wrapper here shouldn't have `isFocused`. I thought we wanted some top level wrapper render props to resolve https://github.com/adobe/react-spectrum/issues/4429?

Reply:
I'll double check this. I was also unsure about this change, so I put them all in the same commit to be easily reverted if needed.

Reply:
I think the plan is to have a separate wrapper that can be used instead, so we don't need it on every component.